activerecord/lib/active_record/connection_adapters/mysql2_adapter.rb

```diff
@@ -14,14 +14,6 @@ def mysql2_connection(config)
       config[:username] = 'root' if config[:username].nil?
       config[:flags] ||= 0
 
-      if Mysql2::Client.const_defined? :FOUND_ROWS
-        if config[:flags].kind_of? Array
-          config[:flags].push "FOUND_ROWS".freeze
-        else
-          config[:flags] |= Mysql2::Client::FOUND_ROWS
```

Comment:
Currently mysql2 adapter always set `CLIENT_FOUND_ROWS` and there is no way to avoid it.
If we expect natural *affected* rows, do we have any way?